smartsim/_core/mli/comm/channel/channel.py

```diff
@@ -46,9 +46,10 @@ def send(self, value: bytes) -> None:
         :param value: The value to send"""
 
     @abstractmethod
-    def recv(self) -> t.List[bytes]:
+    def recv(self, timeout: int = 0) -> t.List[bytes]:
```

Comment:
Not this line, but can you do me a favor and change `Receieve` to `Receive` please :)

Reply:
Fixed misspelled word 5x